cd-db/anon_db.sql

```diff
@@ -0,0 +1,43 @@
+\c cp-users-development
+UPDATE sys_user SET
+  salt='6a43b460',
+  pass='8d8537a931321e9c1fbcc493da36b09888338d4615341016498fd40926b7e115f77c8676f6c06e391415265688658f1d8380bc6f9b45cd33572479e04ec1246f';
+UPDATE sys_user set email = 'testmail+user' || id || '@example.com';
+UPDATE sys_user set nick = email;
+UPDATE sys_user SET
+  nick = 'NameyFace' ,
+  name = 'Namey McNameFace' ,
+  first_name = 'Namey' ,
+  last_name = 'McNameFace' ,
+  phone = translate(phone, '0123456789','9999999999');
+TRUNCATE cd_oauth2;
+UPDATE cd_profiles SET name = 'Namey McNameFace';
+UPDATE cd_profiles SET email = 'testmail+user' || user_id || '@example.com';
+UPDATE cd_profiles SET alias = 'NameyFace';
+UPDATE cd_profiles SET
+  address = '123 Fake St., Over There Ave., End of the Road',
+  phone = translate(phone, '0123456789','9999999999');
+UPDATE cd_profiles SET avatar = NULL, parent_invites = NULL, ninja_invites = NULL, twitter = NULL, linkedin = NULL;
+UPDATE sys_login SET email = 'testmail+user' || sys_login.user || '@example.com', token = NULL;
+UPDATE sys_login SET nick = email;
+UPDATE sys_reset SET nick = 'testmail+user' || sys_reset.user || '@example.com';
+UPDATE cd_agreements SET full_name='Namey McNameFace', ip_address='127.0.0.1';
+
+\c cp-dojos-development
+UPDATE cd_dojoleads SET application = jsonb_set(application::jsonb, '{championDetails, email}'::text[], ('"testmail+user' || user_id || '@example.com"')::jsonb);
```

Comment:
ChampionDetails doesn't exists anymore
You need  "champion", and "venue" at least

Reply:
so what those the data look like?

Reply:
Will commit to fix, it's easier than dumping a json schema